tests/behat/fixtures/d7/composer.json

```diff
@@ -24,6 +25,7 @@
         "drupal/wysiwyg": "^2.5",
         "library/ckeditor": "^4.9"
     },
+    "minimum-stability": "dev",
```

Comment:
please add ` "prefer-stable": true` below if you are adding this one.

Reply:
but why do we need this at all? field collection module is a stable module

Reply:
It is beta version only.